### PR TITLE
Add spec to check that Akka.Cluster.Metrics can be started automatically

### PR DESCRIPTION
**Pull request by Arkatufus**

### CODE REVIEW

File: src/contrib/cluster/Akka.Cluster.Metrics.Tests/ClusterMetricsAutostartSpec.cs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        private readonly ClusterMetricsView _metricsView;
+        
+        private int MetricsNodeCount => _metricsView.ClusterMetrics.Count;
+
+        /// <summary>
+        /// This is a single node test.
+        /// </summary>
+        private const int NodeCount = 1;
+        private static readonly Config Config = ConfigurationFactory.ParseString(@"
+akka.extensions = [""Akka.Management.Cluster.Bootstrap.ClusterBootstrapProvider, Akka.Management.Cluster.Bootstrap""]
```
Comment (Aaronontheweb) [inline src/contrib/cluster/Akka.Cluster.Metrics.Tests/ClusterMetricsAutostartSpec.cs:32]: Isn't this the wrong extension?

Comment (Arkatufus) [reply src/contrib/cluster/Akka.Cluster.Metrics.Tests/ClusterMetricsAutostartSpec.cs]: Yes, it is, fixed it.

File: docs/articles/clustering/cluster-metrics.md [hunk 2]
```diff
@@ -128,6 +128,20 @@ Custom metrics collector implementation class must be specified in the `akka.clu
...
+```hocon
+akka {
+    extensions = ["Akka.Cluster.Metrics.ClusterMetricsExtensionProvider, Akka.Cluster.Metrics"]
+    actor.provider = "cluster"
+    cluster.metrics.collector {
+        provider = ["Akka.Cluster.Metrics.Collectors.DefaultCollector, Akka.Cluster.Metrics"]
+    }  
+}
+```
+These settings will auto-start Akka.Cluster.Metrics using the default memory and CPU metrics collectors.
```
Comment (Aaronontheweb) [inline docs/articles/clustering/cluster-metrics.md:142]: LGTM

File: src/contrib/cluster/Akka.Cluster.Metrics.Tests/ClusterMetricsAutostartSpec.cs [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+}}
+");
+        
+        public ClusterMetricsAutostartSpec(ITestOutputHelper output)
+            : base(Config, output)
+        {
+        }
+
+        [Fact]
+        public void Metrics_extension_Should_autostart_if_added_to_akka_extensions()
```
Comment (Aaronontheweb) [inline src/contrib/cluster/Akka.Cluster.Metrics.Tests/ClusterMetricsAutostartSpec.cs:45]: LGTM

**Review by Aaronontheweb — APPROVED** [review]
LGTM once you make the markdown linter happy ;)

**Pull request merged**